src/sql/pg/create.sql

```diff
@@ -1,7 +1,7 @@
 CREATE TABLE IF NOT EXISTS "L_ACCOUNTS" (
   "ACCOUNT_ID" SERIAL NOT NULL,
   "NAME" CHARACTER VARYING(255) NOT NULL,
-  "BALANCE" DECIMAL(32,16) NOT NULL CHECK
+  "BALANCE" DECIMAL(32,16) DEFAULT 0 NOT NULL CHECK
     ("BALANCE" >= "MINIMUM_ALLOWED_BALANCE"),
```

Comment:
Probably. I was trying to figure out if this was captured in the db schema that Chris has been implementing. I can't find any info on it, so I'll add that constraint now.